documents/src/pages/elements/text-field.md

```diff
@@ -48,9 +48,118 @@ p {
 
 ::
 
-`ef-text-field` is a form element for text.
+`ef-text-field` is a form element for text. This basic text input enables the user to type a small amount of text and submit it into Workspace.
 
-## Usage
+## Design Usage Rules
+
+### Content principles
+
+* Users should understand the state of the field quickly and easily
+* Placeholder text should not contain instructions. If these are needed supportive text should be provided
+* Label text can wrap onto multiple lines if needed. Avoid truncating, where possible  
+* Supportive  text should be used only where needed
+
+### Formatting
+
+Form labels should be visible and there are 3 variants:
+
+#### Stacked
```

Comment:
- Update Sandbox examples in 'Formating' to match the Figma component styling and spacing https://www.figma.com/file/YeKQAvy9EGQyggtae5tvqw/HALO---LIGHT----UI-COMPONENT-LIBRARY?node-id=22890%3A6832&t=53RHf2i3CPQdjW74-4
- Remove placeholder text from the 'Formatting' examples and add it as supportive text (to match the Figma asset)

Reply:
> * Update Sandbox examples in 'Formating' to match the Figma component styling and spacing https://www.figma.com/file/YeKQAvy9EGQyggtae5tvqw/HALO---LIGHT----UI-COMPONENT-LIBRARY?node-id=22890%3A6832&t=53RHf2i3CPQdjW74-4

Could you be more specific on styling and spacing? I could only see one parity: labels' font-weight should be 600.

> * Remove placeholder text from the 'Formatting' examples and add it as supportive text (to match the Figma asset)

Didn't we agree that placeholders would be used as supportive texts?

Reply:
You should be able to view the details by selecting the element in question in Figma. The form label is the Body 4 style.
Spacing is recorded in the auto layouts by moving your mouse to the adjacent item with the 'Option' key pressed

![Screenshot 2023-02-15 at 09 58 26](https://user-images.githubusercontent.com/120190925/218995657-c26e912c-08e0-4284-8871-c8cfd570cc96.png)
![Screenshot 2023-02-15 at 09 58 44](https://user-images.githubusercontent.com/120190925/218995661-12f4be05-b940-425b-a6d7-15da85967969.png)
![Screenshot 2023-02-15 at 09 58 51](https://user-images.githubusercontent.com/120190925/218995663-e2cd3b6d-b964-4ea5-9481-312134b466be.png)
![Screenshot 2023-02-15 at 09 59 02](https://user-images.githubusercontent.com/120190925/218995668-a9199555-4126-42d9-a903-6bcc6850e094.png)

Reply:
The placeholder conversation was before the party work when we were just looking at a11y in isolation.
We should still have a placeholder example somewhere but the default parity asset uses supportive text

Reply:
* Thanks for the clarification on styling and spacing. Will take care of it.
* for supportive text, does this mean we will apply this change to "Formating" section only?  Also, would a `<span>` with `aria-describedby` be a proper supportive text?

Reply:
styling and spacing for labels updated.

Reply:
> * Thanks for the clarification on styling and spacing. Will take care of it.
> * for supportive text, does this mean we will apply this change to "Formating" section only?  Also, would a `<span>` with `aria-describedby` be a proper supportive text?

Probably - how does it read out in a screen reader?

Reply:
> Probably - how does it read out in a screen reader?

Let me try it out. 

> for supportive text, does this mean we will apply this change to "Formating" section only? 

As you didn't say anything about it, I'm taking it as a "yes".

Reply:
yup - formating section only

Reply:
## Summary
I think it works find providing the experience with VO on Safari is not ideal yet still acceptable.

## Test Result

### VO Chrome macOS 🎉
> Full name, edit text [short pause] Your name as shown on your passport, you're currently on a text field. To enter text to this field, type.

### VO Safari macOS 👌🏼
> full name, edit text, more content available.

to access more conent, press ctrl + option + command + /. Check the screenshots for better understanding. 
<img width="1593" alt="more content available" src="https://user-images.githubusercontent.com/117723407/220550548-e2df3199-502d-4b02-9010-86f44cff56b4.png">
<img width="1612" alt="more content display" src="https://user-images.githubusercontent.com/117723407/220550588-9ec36740-3c18-4d7f-aa34-ec777f2a05f1.png">

### NVDA Windows 🎉
> [typewriting sound] full name, edit, your name as shown on your passport